integration_tests/dbt_project_evaluator/dbt_project.yml

```diff
@@ -0,0 +1,133 @@
+          +enabled: false
+      fct_model_6:
+        +materialized: table
+  dbt_project_evaluator:
+    marts:
+      tests:
+        fct_test_coverage:
+          # materialize as a table to ensure SQL query runs successfully
+          +materialized: table
+      documentation:
+        fct_documentation_coverage:
+          # materialize as a table to ensure SQL query runs successfully
+          +materialized: table
+      structure:
+        # Overridden by trino__fct_model_naming_conventions
+        fct_model_naming_conventions:
+          +enabled: false
+      dag:
+        # Overridden by trino__fct_model_fanout
+        fct_model_fanout:
+          +enabled: false
+
+tests:
+  dbt_project_evaluator:
+    +enabled: false
+  dbt_project_evaluator_integration_tests:
+    dbt_project_evaluator_schema_tests:
+      unique_int_all_dag_relationships_path:
+        # Grouping by expressions of type ARRAY is not allowed for BigQuery
+        +enabled: "{{ false if target.type in ['bigquery'] else true }}"
+    dag:
+      # Overridden by trino__equality_fct_duplicate_sources
+      equality_fct_duplicate_sources:
+        +enabled: false
+      # Overridden by trino__equality_fct_model_fanout
+      equality_fct_model_fanout:
+        +enabled: false
+    structure:
+      # Overridden by trino__equality_fct_model_naming_conventions
+      equality_fct_model_naming_conventions:
+        +enabled: false
+    performance:
+      # Overridden by trino__equality_fct_chained_views_dependencies
+      equality_fct_chained_views_dependencies:
+        +enabled: false
+
+seeds:
+  dbt_project_evaluator:
+    dbt_project_evaluator_exceptions:
+      +enabled: false
+  dbt_project_evaluator_integration_tests:
+    # Overridden by trino__dbt_project_evaluator_exceptions
+    dbt_project_evaluator_exceptions:
+      +enabled: false
+    tests:
+      # Overridden by trino__test_fct_test_coverage
+      test_fct_test_coverage:
+        +enabled: false
+    docs:
+      # Overridden by trino__test_fct_documentation_coverage
+      test_fct_documentation_coverage:
+        +enabled: false
+    structure:
+      # Overridden by trino__test_fct_model_naming_conventions
+      test_fct_model_naming_conventions:
+        +enabled: false
+    dag:
+      # Overridden by trino__test_fct_model_fanout
+      test_fct_model_fanout:
+        +enabled: false
+    performance:
+      # Overridden by trino__test_fct_chained_views_dependencies
+      test_fct_chained_views_dependencies:
+        +enabled: false
+
+sources:
+  dbt_project_evaluator_integration_tests:
+    staging:
+      source_1:
+        # Overridden by trino__source_4
+        source_3:
+          +enabled: false
+
+vars:
+  # -- DAG variables --
+  models_fanout_threshold: 3
+
+  exclude_paths_from_project: ["/to_exclude/","trino__source_4.trino__table_6"]
+
+  # -- Performance variables --
+  chained_views_threshold: 3
+
+  # -- Execution variables --
+  max_depth_dag: 3
```

Comment:
is it used?

Reply:
Yes it is used in [recursive_dag](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L110) macro

Reply:
how about `chained_views_threshold` ?

Reply:
used also in `recursive_dag` macro